src/Symfony/Bundle/TwigBundle/Resources/views/Exception/logs.html.twig

```diff
@@ -22,6 +22,12 @@
                 </td>
             {% endif %}
             <td>{{ log.message|format_log_message(log.context) }}</td>
+            {# hide the context of the 'event' channel because it only displays duplicated information #}
+            {% if log.context ?? false and log.channel != 'event' %}
+                <div class="text-muted">
+                    {{ profiler_dump(log.context) }}
```

Comment:
this will indeed create a hard dependency on WebProfilerBundle in TwigBundle (which will break things in environments where WebProfilerBundle is not available)

Reply:
How can we solve this? Maybe we can replace `profiler_dump` by `yaml_dump`? https://symfony.com/doc/current/reference/twig_reference.html#yaml-dump